toolchain/c-toolchain.lisp

```diff
@@ -57,18 +57,25 @@
 
 ;;; C support
 
-(defparameter *cc*
-  #+(or cygwin (not windows)) "cc"
-  #+(and windows (not cygwin)) "gcc")
+(defparameter *default-cc*
+  ;; the copy-seq ensures we have a fresh string
+  ;; which allows for safe testing with EQ for someone else
+  ;; having set it.
+  (copy-seq
+   (or (getenv "CC")
+       #+(or cygwin (not windows)) "cc"
+       #+(and windows (not cygwin)) "gcc")))
 
-(defparameter *cc-flags*
-  (append
-   ;; For MacPorts
-   #+darwin (list "-I" "/opt/local/include/")
-   ;; ECL internal flags
-   #+ecl (parse-command-flags c::*cc-flags*)
-   ;; FreeBSD non-base header files
-   #+freebsd (list "-I" "/usr/local/include/")))
+(defparameter *cc* *default-cc*)
```

Comment:
I didn't do this because that changes the externally visible behavior (i.e. I don't know that someone out there isn't reading from *cc* or *cc-flags* to see what the default parameters are)